libgdf/include/memory.h

```diff
@@ -18,17 +18,17 @@
  * @brief Device Memory Manager public interface. 
  * 
  * Efficient allocation, deallocation and tracking of GPU memory.
- * ---------------------------------------------------------------------------**/
+ * --------------------------------------------------------------------------**/
 
-#pragma once
+#pragma once 
```

Comment:
I think we agreed to use standard `#ifndef`s instead of the non-standard `#pragma once`

Reply:
I agree, but I can't in this file, because it's parsed by CFFI, which barfs on any macro, but seems to ignore pragma.

Reply:
Notice I used `#ifndef` in `rmm.h`.

Reply:
@jrhemstad Can you please approve this so I can remove the "Needs Rework" label?